### DISCUSSION TITLE
Vuu contribution demo and Q&A

### DISCUSSION THREAD

**Discussion by ColinEberhardt** [finos/technical-oversight-committee#40] [Polls]
The team who created, Vuu, which is being [considered for contribution](https://github.com/finos/community/issues/206) have offered to give a demo and answer questions. I'd like to determine which dates work best for everyone. 

If multiple times (or all of them) work for you, please leave a comment.

**Comment by jonfreedman** [comment]
I'm in New York this week so can't do Fri 7th

**Reply by ColinEberhardt** [reply]
Thanks for the heads-up Jon
